packages/ra-core/src/reducer/index.ts

```diff
@@ -13,7 +12,6 @@ export default (customReducers, locale, messages, history) =>
     combineReducers({
         admin,
         i18n: i18nReducer(locale, messages),
-        form: formReducer,
```

Comment:
this should be documented in custom apps and upgrade

Reply:
Done